lib/content-services/src/lib/upload/components/upload-version-button.component.ts

```diff
@@ -35,6 +35,10 @@ export class UploadVersionButtonComponent extends UploadButtonComponent implemen
     @Input()
     node: Node;
 
+    /** (**Required**) The drag&dropped new file to be versioned. */
+    @Input()
+    newFileVersion: File;
```

Comment:
why creating second property and then assigning it to the `file`? I suggest using base.file for that